### DISCUSSION TITLE
OpenGraph integration

### DISCUSSION THREAD

**Discussion by ayoahha** [requarks/wiki#6519] [Help / Questions] [answered]
Hi everybody
It seems that i do not have any opengraph visualisation when i post a link of my wiki...

Is there any reason, or should i just add some tags or something similar to a HTML field ?

Or maybe opengraph cannot be integrated yet ?

Many thanks for your help
Ayo

---
**Notes** :
Wiki.js version: 2.5.295
Database engine: PostgreSQL 11.18

**Comment by ayoahha** [comment]
Snif, nobody can answer the question ?

**Comment by NGPixel** [comment, marked as answer]
Submit feature requests on https://requarks.canny.io/wiki
